dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/AbstractYarnTask.java

```diff
@@ -107,7 +110,7 @@ public void cancelApplication() throws TaskException {
      */
     @Override
     public List<String> getApplicationIds() throws TaskException {
-        return LogUtils.getAppIdsFromLogFile(taskRequest.getLogPath(), logger);
+        return LogUtils.getAppIds(taskRequest.getLogPath(), taskRequest.getAppInfoPath(), PropertyUtils.getString(APPID_COLLECT, "log"));
```

Comment:
It is recommended that the log string be added to the constant.

Reply:
Thanks for reminding!